package.json

```diff
@@ -47,7 +47,7 @@
     "@vitejs/plugin-react": "^4.2.1",
     "@yext/pages": "^1.2.0-beta.3",
     "@yext/pages-components": "^1.0.0-rc.9",
-    "@yext/visual-editor": "0.0.9",
+    "@yext/visual-editor": "0.0.10",
```

Comment:
Do we want a caret here? It wasn't there before.

Reply:
no because if we release something that is breaking new starters will break. this is just for security for now